MAINTAINERS.md

```diff
@@ -1,18 +1,122 @@
+
+From [Apache’s versioning doc](https://commons.apache.org/releases/versioning.html) : 
+
+A release number is comprised of 3 components: the major release number, the minor release number, and an optional point release number. Here is a sample release number:
+2.0.4
+and it can be broken into three parts:
+
+    * major release: 2
+    * minor release: 0
+    * point release: 4
+
+The next release of this component would increment the appropriate part of the release number, depending on the type of release (major, minor, or point). For example, a subsequent minor release would be version 2.1, or a subsequent major release would be 3.0.
+Note that release numbers are composed of three _integers_, not three digits. Hence if the current release is 3.9.4, the next minor release is 3.10.0.
+
+OpenSearch and OpenSearch Dashboards will release major version together.   They will NOT synchronize minor release — whenever the team feels they’re ready to release a minor version or patch (modulo the schedule above), they should release.   
+
+What we guarantee is that any major release of NotKibana is compatible with the same major release of NotElasticserach.  For example:   3.2.1 of NotKibana will work with 3.0.4 of NotElasticsearch, but 2.3.1 of NotKibana is not guaranteed to work with 3.0.4 of NotElasticsearch
+
+## Breaking Changes and Backwards Compatibility
+
+We do not release breaking changes except in major releases.  This means you.  *Paddington Hard Stare*.  The goal for 1.x is the compatible with Elasticsearch 7.x and 6.8., so the first ime we'd introduce breaking changes is 2.0
+
+## Branches
+
+### Primary Branches 
+Currently, OpenSearch has three branches: 1.0, 1.x, Main
+
+*Main* is our next major release. This is the location that all merges should take place. It's going to moving fast and dynamic in there.
+
+*1.x* Is our next minor release. Once something gets merged into main, you can chose to backport it to 1.x.
+
+*1.0* is our current release. In between minor releases, only hotfixes (security and otherwise) would get backported to 1.0.
+
+When you review a PR apply the next major version label (e.g., 2.0) and if accepted, merge it into Main. If the requestor thinks it should be backported and released with 1.0, they should open a separate PR and the reviewer will label it with the 1.x label. Then we'll merge the new PRs to the 1.x branch. If you don't know what release a requestor wants to target, ask!
+
+In the nearish future we'd like to see nightly of all three branches (1.0, 1.x and 2.0) so we can rapidly find regressions, but I'll need to hash that out with infra.
+
+Why do it this way?
+Because it lets main evolve quickly, while letting us be a little more circumspect about 1.x and 1.0. It'll also make it easier for us to release if everything is clearly tagged (not having clear tagging made this release kind of a potchke).
+
```

Comment:
You need to talk about `-rc1`, and `-beta1`.

Reply:
I'm not expecting that to be a going concern in ~6 weeks.  If you think it's useful for past context, I can add it.